viewer/src/components/PlayerInfo.vue

```diff
@@ -235,6 +236,12 @@ export default class PlayerInfo extends Vue {
     return factions[this.player.faction].name;
   }
 
+  recentAction(i: number): boolean {
+    const action = this.player.actions[i];
+    const moves = this.$store.getters["gaiaViewer/recentCommands"];
+    return moves.some((c) => c.command == Command.Special && c.args[0] === action.rewards);
+  }
```

Comment:
For better ⚡️⚡️, add a getter `recentSpecialActions` in `store.ts` then the calculation will only be done once for all the recent special actions of all players.

Also if multiple players have the `=>+4pw` tech tile, I think that one player activating the special action will trigger it for everyone.